_generated/CondaAuthenticateV0/condaauthenticatemain.ts

```diff
@@ -8,6 +8,7 @@ async function main(): Promise<void> {
         const localAccesstoken = tl.getVariable('System.AccessToken');
         tl.debug(tl.loc('AddingAuthChannel', 'ARTIFACTS_CONDA_TOKEN'));
         tl.setVariable('ARTIFACTS_CONDA_TOKEN', localAccesstoken);
+        tl.setSecret(localAccesstoken);
```

Comment:
Out of my own curiosity, are we setting the secret for the first time with this PR? If were adding the setSecret, and not removing anything, the old behavior will still be present. Is that expected?

Reply:
I see now that this isn't actually setting the secret to a variable. So all we need here is to mark the var as a secret, which is what this line is doing.